test/compliance/generate_fbank_data.py

```diff
@@ -0,0 +1,174 @@
+# where the space separates an id from a wav file.
+SCP_PATH = 'scp:/scratch/jamarshon/downloads/a.scp'
+# The directory to which the stft features will be written to.
+OUTPUT_DIR = 'ark:/scratch/jamarshon/audio/test/assets/kaldi/'
+
+# The number of samples inside the input wave file read from `SCP_PATH`
+WAV_LEN = 20
+
+# How many output files should be generated.
+NUMBER_OF_OUTPUTS = 100
+
+VERBOSE = False
+
+
+def generate_rand_boolean():
+    # Generates a random boolean ('true', 'false')
+    return 'true' if random.randint(0, 1) else 'false'
+
+
+def generate_rand_window_type():
+    # Generates a random window type
+    return torchaudio.compliance.kaldi.WINDOWS[random.randint(0, len(torchaudio.compliance.kaldi.WINDOWS) - 1)]
+
+
+def run():
+    for i in range(NUMBER_OF_OUTPUTS):
+        try:
+            nyquist = 16000 // 2
+            high_freq = random.randint(1, nyquist)
+            low_freq = random.randint(0, high_freq - 1)
+            vtln_low = random.randint(low_freq + 1, high_freq - 1)
+            vtln_high = random.randint(vtln_low + 1, high_freq - 1)
+            vtln_warp_factor = random.uniform(0.0, 10.0) if random.random() < 0.3 else 1.0
+
+        except Exception:
+            pass
+
+        if not ((0.0 <= low_freq < nyquist) and (0.0 < high_freq <= nyquist) and (low_freq < high_freq)):
+            continue
+        if not (vtln_warp_factor == 1.0 or ((low_freq < vtln_low < high_freq) and
+                                            (0.0 < vtln_high < high_freq) and (vtln_low < vtln_high))):
+            continue
+
+        inputs = {
+            'blackman_coeff': '%.4f' % (random.random() * 5),
+            'energy_floor': '%.4f' % (random.random() * 5),
+            'frame_length': '%.4f' % (float(random.randint(3, WAV_LEN - 1)) / 16000 * 1000),
+            'frame_shift': '%.4f' % (float(random.randint(1, WAV_LEN - 1)) / 16000 * 1000),
+            'high_freq': str(high_freq),
+            'htk_compat': generate_rand_boolean(),
+            'low_freq': str(low_freq),
+            'num_mel_bins': str(random.randint(4, 8)),
+            'preemphasis_coefficient': '%.2f' % random.random(),
+            'raw_energy': generate_rand_boolean(),
+            'remove_dc_offset': generate_rand_boolean(),
+            'round_to_power_of_two': generate_rand_boolean(),
+            'snip_edges': generate_rand_boolean(),
+            'subtract_mean': generate_rand_boolean(),
+            'use_energy': generate_rand_boolean(),
+            'use_log_fbank': generate_rand_boolean(),
+            'use_power': generate_rand_boolean(),
+            'vtln_high': str(vtln_high),
+            'vtln_low': str(vtln_low),
+            'vtln_warp': '%.4f' % (vtln_warp_factor),
+            'window_type': generate_rand_window_type()
+        }
+
+        fn = 'fbank-' + ('-'.join(list(inputs.values())))
+        out_fn = OUTPUT_DIR + fn + '.ark'
+        arg = [
```

Comment:
You should be able to pass the inputs directly to "arg" without the parameter "inputs".